tests/eval/mod.rs

```diff
@@ -150,7 +149,7 @@ fn or_short_circuit() -> LangResult<'static, ()> {
     panic_after(Duration::from_secs(1), || {
         let input = include_str!("or_short_circuit.pj");
         let term = run(input)?;
-        assert_eq!(Term::Lit(Literal::Bool(true)), term);
+        assert_eq!(Term::Lit(1), term);
```

Comment:
maybe we could implement `From<i64>`, `From<bool>` and `From<()>` for `Term` in order to have some "typedness" in our tests.

Reply:
Added some `From` impls